src/pseudo_people/noise.py

```diff
@@ -0,0 +1,41 @@
+import pandas as pd
+
+from pseudo_people.configuration import NoiseConfiguration
+from pseudo_people.entities import COLUMNS_METADATA, ROW_NOISE_TYPES, Form
+
+
+def noise_form(
+    form: Form,
+    form_data: pd.DataFrame,
+    noise_config: NoiseConfiguration = None,
```

Comment:
Need to create an actual default `NoiseConfiguration` object to use

Reply:
We should also have a default randomness stream that will be used?

Reply:
Great point. I've updated this to include that.